rekolekcje-engine/src/test/java/pl/oaza/warszawa/dor/rekolekcje/api/participants/ParticipantBirthDateTest.java

```diff
@@ -11,7 +12,7 @@
   @Test
   public void shouldRetrievedParticipantHaveBirthDate(){
     //given
-    ParticipantDTO participantDTO = ParticipantsTestData.participantWithFullData;
+    ParticipantDTO participantDTO = ParticipantFactory.withFullData(null);
```

Comment:
Nie podoba mi się przekazywanie nulla jako argumentu

Reply:
Ten null jest celowo. Mógłbym przeciążyć metodę, ale chodzi o to, żeby wyraźnie zaznaczyć, że tworzymy requesta bez id, a jest ono zwracane przez backend.